src/cmd/services/m3dbnode/config/pooling.go

```diff
@@ -252,6 +253,9 @@ type PoolingPolicy struct {
 	// The initial alloc size for a block.
 	BlockAllocSize *int `yaml:"blockAllocSize"`
 
+	// The thrift bytes pool max bytes slice allocation for a single binary field.
+	ThriftBytesPoolMaxAllocSize *int `yaml:"thriftBytesPoolMaxAllocSize"`
```

Comment:
Its not really a max is it right? its always this size?

Reply:
The capacity is always this size, but the actual length/size is determined by the length of the bytes being copied. 

Since we suffix it with "Alloc" I'm not opposed to just calling it `AllocSize` and dropping the `Max`.